src/components/Article.js

```diff
@@ -43,22 +48,32 @@ export const Article = ({ lang, title }) => {
     window.location.hash = `/quickfacts/${lang}/${title}`
   }
 
-  if (!article) {
-    return 'Loading...'
-  }
-
   const hasImage = !!article.imageUrl
 
+  const pagerEvent = {
+    nextPage: () => {
+      const sectionLength = article.sections.length
+      const nextSection = currentSection + 1
+
+      if (nextSection < sectionLength) setCurrentSection(nextSection)
+      else setCurrentSection(0)
+    },
+    prevPage: () => {
+      const prevSection = currentSection - 1
+      if (prevSection >= 0) setCurrentSection(prevSection)
+    }
+  }
+
   return (
     <Fragment>
-      <Pager>
+      <Pager event={pagerEvent}>
         <div class='page article'>
           { hasImage && <div class='lead-image' style={{ backgroundImage: `url(${article.imageUrl})` }} /> }
           <div class={'card' + (hasImage ? ' with-image' : '')}>
             <div class='title' dangerouslySetInnerHTML={{ __html: article.title }} />
-            { article.description && (
+            { article.sections[currentSection].description && (
```

Comment:
_the `title` always keep the same, the `description` shows the `lead description` or the section `header text`_

the actual design of this will be defined later.